src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -185,16 +185,47 @@ public class UnsolvedSymbolVisitor extends ModifierVisitor<Void> {
   private final Map<@ClassGetSimpleName String, List<@ClassGetSimpleName String>>
       classToItsUnsolvedInterface = new HashMap<>();
 
+  /** List of target methods as specified by users. */
+  private final Set<String> targetMethodsNames;
+
+  /**
+   * Fields and methods that could be called inside the target methods. We call them potential-used
+   * because the usage check is simply based on the simple names of those members.
+   */
+  private final Set<String> potentialUsedMembers = new HashSet<>();
+
+  /**
+   * Check whether the visitor is inside the declaration of a target method. Symbols inside the
+   * declarations of target methods will be solved if they have one of the following types:
+   * ClassOrInterfaceType, Parameters, VariableDeclarator, MethodCallExpr, FieldAccessExpr,
+   * ExplicitConstructorInvocationStmt, NameExpr, MethodDeclaration, and ObjectCreationExpr.
+   */
+  private boolean insideTargetMethod = false;
+
+  /**
+   * Check whether the visitor is inside the declaration of a member that could be used by the
+   * target methods. Symbols inside the declarations of potentially-used members will be solved if
+   * they have one of the following types: ClassOrInterfaceType, Parameters, and VariableDeclarator.
+   */
+  private boolean insidePotentialUsedMember = false;
+
+  /** The qualified name of the current class. */
+  private String currentClassQualifiedName = "";
+
   /**
    * Create a new UnsolvedSymbolVisitor instance
    *
    * @param rootDirectory the root directory of the input files
    * @param setOfExistingFiles the set of existing files in the input codebase
+   * @param targetMethodNames the list of target methods as specified by the user.
```

Comment:
Is this signatures (as specified by the user) or is it simple names? Clarify this throughout the documentation.

Reply:
`UnsolvedSymbolVisitor` makes this clear around line 700 that the intention is for these to be signatures, not names. Please change the documentation to reflect that.